challengeFive.js

```diff
@@ -0,0 +1,11 @@
+function hideNumber(number){
+  const cardNumber = number.toString();
+  const hiddenNumber = "*".repeat(cardNumber.length - 4)
+   return  hiddenNumber + cardNumber.slice(- 4) 
```

Comment:
Formatting issue here with the return statement and also the spaces between `return` and `hiddenNumber`. Also the `- 4` within the slice should be `-4`